customizations/README.md

```diff
@@ -447,24 +447,56 @@ In order to use Google Cloud Storage custom buckets you need to:
 
 2. Configure the required [CORS settings](#requirements).
 
-3. Create a [custom Service account](#custom-service-account).
+3. Add the following lines to your `customizations.yaml` and replace the `<values>` with your own settings:
 
-4. Grant this service account with the following role (in addition to the buckets access): `roles/iam.serviceAccountTokenCreator`.
+    ```yaml
+    appConfigValues:
+      storageProvider: "gcp"
+      importBucket: <import_bucket_name>
+      workspaceImportsBucket: <client_bucket_name>
+      workspaceImportsPublic: <false|true>
+      workspaceThumbnailsBucket: <thumbnails_bucket_name>
+      workspaceThumbnailsPublic: <false|true>
+      googleCloudStorageProjectId: <gcp_project_id>
+    ```
 
-   > :warning: We don't recommend grating this role at project IAM level, but instead at the Service Account permissions level (IAM > Service Accounts > `your_service_account` > Permissions).
+4. Select a Service Account to use. You can use a [custom Service account for all things](#custom-service-account),  the Service account provided by Carto in your package or create a dedicated Service account only for this buckets.
 
-5. Add the following lines to your `customizations.yaml` and replace the `<values>` with your own settings:
+   > :warning: Make sure that the selected service account had  `roles/iam.serviceAccountTokenCreator` role in the GCP project that created it. We don't recommend grating this role at project IAM level, but instead at the Service Account permissions level (IAM > Service Accounts > `your_service_account` > Permissions).
```

Comment:
```suggestion
5.  Grant the selected Service Account with the role `roles/iam.serviceAccountTokenCreator` in the GCP project where it was created.
      > :warning: We don't recommend grating this role at project IAM level, but instead at the Service Account permissions level (IAM > Service Accounts > `your_service_account` > Permissions).
```

I think it will be more clear if the IAM role is set in a dedicated step, rather than in a warning.